core/api/package.json

```diff
@@ -161,6 +161,7 @@
     "@typescript-eslint/parser": "^7.7.0",
     "@xascode/tscpaths": "0.1.4",
     "axios-mock-adapter": "^1.22.0",
+    "child_process": "^1.0.2",
```

Comment:
last update 8 years ago https://www.npmjs.com/package/child_process

Reply:
doesn't matter? it's on dev only